blackbox/docs/general/builtins/scalar.rst

```diff
@@ -1856,6 +1856,20 @@ Example::
     SELECT 1 row in set (... sec)
 
 
+``current_database()``
+----------------------
+
+The ``current_database`` function returns the name of the current database,
+which in CrateDB will always be ``CrateDB``::
+
+    cr> select current_database();
+    +--------------------+
+    | current_database() |
+    +--------------------+
+    | CrateDB            |
```

Comment:
should we maybe use lower_case instead?

Reply:
/cc @joemoe  any opinion on this?

Reply:
in postgres the user can define it, right?

Reply:
@seut changed it to `crate`